bound/kt/src/main/kotlin/tbdex/sdk/web5/PresentationDefinition.kt

```diff
@@ -0,0 +1,40 @@
+package tbdex.sdk.web5
+
+import tbdex.sdk.rust.InputDescriptorData as RustCoreInputDescriptor
+import tbdex.sdk.rust.PresentationDefinition as RustCorePresentationDefinition
+import tbdex.sdk.rust.PresentationDefinitionData as RustCorePresentationDefinitionData
+
+typealias InputDescriptor = RustCoreInputDescriptor
+
+class PresentationDefinition {
+    val id: String
+    val name: String?
+    val purpose: String?
+    val inputDescriptors: List<InputDescriptor>
+
+    val rustCorePresentationDefinition: RustCorePresentationDefinition
```

Comment:
we have to do this because of the json thing?

Reply:
We have to do this one because of the `select_credentials` instance method, for the JSON thing we'll have to redefine the data structures at the kt layer (we don't currently need to do this because the inner rust code doesn't yet support it) -- see the `Offering.kt` as an example of this